core-java/src/main/java/eu/neverblink/jelly/core/JellyOptions.java

```diff
@@ -237,7 +237,7 @@ private static void checkBaseCompatibility(RdfStreamOptions requestedOptions, Rd
      *
      * @throws RdfProtoDeserializationError if the table size is not within the supported range.
      */
-    private static void checkTableSize(String name, int size, int supportedSize, int minSize) {
+    public static void checkTableSize(String name, int size, int supportedSize, int minSize) {
```

Comment:
Please mark such cases (moving from private to public just for the sake of Java) as `@Internal` or whatever. Maybe we should have an annotation type for this in core? Or reuse something from protobuf-java, if they have it.

Reply:
Internal does not exist, lets leave as is, as after "habibi" it will be possible to generalize it

Reply:
Still, could you make an `@Internal` annotation defined in `coreJava`? This will make it easier for end users to tell what is and isn't part of the public API.

Reply:
Aight and also lets define our own `@Experimental`, for consistency purposes